### PR TITLE
BUG: SYSV settings are ignored

### PR DESCRIPTION
**Pull request by cernst1980**
When setting sysvmsg, sysvsem and sysvshm to value of "new", jail starts without these settings.
The settings were skipped because of "if '1' in x".

Make sure to follow and check these boxes before submitting a PR! Thank you.

- [x] Explain the feature
- [x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: iocage_lib/ioc_start.py [hunk 1]
```diff
@@ -367,6 +367,9 @@ def __start_jail__(self):
             x for x in net
             + [x for x in parameters if '1' in x]
             + [
+                _sysvmsg,
+                _sysvsem,
+                _sysvshm,
```
Comment (skarekrow) [inline iocage_lib/ioc_start.py:373]: Good catch, can we follow the other formatting and move these under the name?

**Review by skarekrow — CHANGES_REQUESTED** [review]

File: iocage_lib/ioc_start.py (hunk 1 above)
Comment (cernst1980) [reply iocage_lib/ioc_start.py:373]: Sorry, I do not get what you mean.
Can you please suggest a specific code change.

File: iocage_lib/ioc_start.py [hunk 2]
```diff
@@ -367,6 +367,9 @@ def __start_jail__(self):
...
                 f'name=ioc-{self.uuid}',
```
Comment (skarekrow) [inline iocage_lib/ioc_start.py]:
```suggestion
                 f'name=ioc-{self.uuid}',
                 _sysvmsg,
                 _sysvsem,
                 _sysvshm,
```
Something similar to that. Not sure why Github is showing that topline change, may be the newline.

**Review by skarekrow — CHANGES_REQUESTED** [review]

File: iocage_lib/ioc_start.py (hunk 1 above)
Comment (skarekrow) [reply iocage_lib/ioc_start.py:373]: Sure remove them from here and look at my suggestion

File: iocage_lib/ioc_start.py [hunk 3]
```diff
@@ -367,7 +367,13 @@ def __start_jail__(self):
...
             + [x for x in parameters if '1' in x]
             + [
-                f'name=ioc-{self.uuid}',
+                _sysvmsg,
+                _sysvsem,
+                _sysvshm,
+                 f'name=ioc-{self.uuid}',
+                 _sysvmsg,
+                 _sysvsem,
+                 _sysvshm,
```
Comment (skarekrow) [inline iocage_lib/ioc_start.py]: From the GH preview it looks like these have an extra whitespace in front? Make sure they line up with host.domainname below

**Review by skarekrow — CHANGES_REQUESTED** [review]

**Review by skarekrow — APPROVED** [review]

**Comment by mkaniewski** [comment]
Could you please merge these changes to 1.0 (stable)?

**Pull request merged**